packages/fxa-settings/src/lib/cache.ts

```diff
@@ -14,6 +14,22 @@ export interface OldSettingsData {
   metricsEnabled?: boolean;
 }
 
+export function getOldSettingsData({
```

Comment:
Without this, I was getting errors in content-server after reset around disallowed keys attempting to be set, like `authAt` and `clientMutationId`.